bedrock/firefox/templates/firefox/firstrun/firstrun-quantum.html

```diff
@@ -37,6 +44,7 @@ <h1>
           {{_('Sign in to your account and we’ll sync the bookmarks, passwords and other great things you’ve saved to Firefox on other devices.')}}
         {% endif %}
         </p>
+      {% endblock %}
         <a href="{{ url('firefox.features.sync') }}" rel="noopener noreferrer" target="_blank">{{_('Learn more about Firefox Accounts')}}</a>
```

Comment:
[From Justin in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1451051#c7), looks like we need to add UTM params to this link for each variant to measure clicks. Or can we already get that from GA/GTM just based on the URL?

Reply:
Doh, I didn't check the link in the bug comment, the UTM params go on the iframe not this link.